discovery/db_discovery.go

```diff
@@ -41,6 +42,20 @@ func NewDBOrchestratorPoolCache(node *core.LivepeerNode) *DBOrchestratorPoolCach
 	return &DBOrchestratorPoolCache{node: node}
 }
 
+func (dbo *DBOrchestratorPoolCache) GetURLs() []*url.URL {
```

Comment:
Needs tests; be sure to cover the error cases as well

Reply:
Added tests